Cue/app/reducers/library.js

```diff
@@ -152,29 +155,25 @@ function library(state: State = initialState, action: Action): State {
   } else if (action.type === 'SHARE_CODE_GENERATED') {
     let deck = decks.find((deck: Deck) => deck.uuid === action.uuid)
     deck.share_code = action.code
+    deck.last_update = new Date()
 
-  } else if (action.type === 'DECK_SYNCED') {
+  } else if (action.type === 'DECK_SYNCED' || action.type === 'DECK_CONFLICT_RESOLVED') {
     let change = action.change
-    let serverDeck = action.serverDeck
-    let deckIndex = decks.findIndex(deck => deck.uuid == change.uuid || deck.uuid == serverDeck.uuid)
+    let updatedDeck = action.updatedDeck
+    let deckIndex = decks.findIndex(deck => deck.uuid == change.uuid || deck.uuid == updatedDeck.uuid)
     if (decks[deckIndex]) {
-      if (change.user_data_version)
+      if (change.user_data_version) {
         decks[deckIndex] = {...decks[deckIndex], user_data_version: change.user_data_version}
-      else
-        decks[deckIndex] = serverDeck
+      } else if (updatedDeck.deleted) {
+        decks.splice(deckIndex,1)
```

Comment:
Isn't this already handled by the `inaccessibleDecks` stuff in `LOADED_LIBRARY`?